tests/functional/firefox/test_ios.py

```diff
@@ -0,0 +1,34 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+from selenium.common.exceptions import TimeoutException
+
+from pages.firefox.ios import IOSPage
+
+
+@pytest.mark.flaky(reruns=1)
```

Comment:
It occurs to me that we should include a reason when marking tests as flaky. It should be safe to use `reason="My reason"` here, and the pytest-rerunfailures plugin could be enhanced to report this reason in the console summary.